src/cdk/create-request-authorizer.ts

```diff
@@ -19,9 +19,15 @@ export function createRequestAuthorizer(
 
   const domainName = getDomainName(stackConfig);
   const functionName = `aws-simple-request-authorizer-${getHash(domainName)}`;
+  const functionNameHash = getHash(functionName);
+
+  const logGroup = new aws_logs.LogGroup(stack, `LogGroup${functionNameHash}`, {
+    retention: aws_logs.RetentionDays.TWO_WEEKS,
+    logGroupName: `/aws/lambda/${functionNameHash}`,
```

Comment:
`/aws/lambda/${functionName}` is more readable

Reply:
I kept the hash in the name because the previous version were generating log groups like `/aws/lambda/GET-Cms-Victor-Bff-Integrator-Lambda-v1-3-4-c9f3c15` and `/aws/lambda/aws-simple-request-authorizer-965d7ef`.

For the request authorizer I'm not sure how the stacks behave / if it would confuse people if the aws account has multiple deployments and all authorizer log data is going to the same log group.

Reply:
But for the other lambda functions it should be safe to omit the hash since the name contains the version

Reply:
Sorry, I was wrong in my comment. `functionName` already has a hash at the end which is constructed of domain parts, so this should not cause any collisions / problems. Changed as suggested.